types/component-emitter/index.d.ts

```diff
@@ -5,13 +5,13 @@
 
 // TypeScript Version: 2.2
 
-interface Emitter {
-    on(event: string, listener: Function): Emitter;
-    once(event: string, listener: Function): Emitter;
-    off(event?: string, listener?: Function): Emitter;
-    emit(event: string, ...args: any[]): Emitter;
-    listeners(event: string): Function[];
-    hasListeners(event: string): boolean;
+interface Emitter<Event = string> {
```

Comment:
Can `Event` be a non-string? If not, this type parameter declaration should be `Event extends string = string`

Reply:
I think an `enum` would be a reasonable type expectation as well